glass/galaxies.py

```diff
@@ -15,10 +15,6 @@
 .. autofunction:: galaxy_shear
 .. autofunction:: gaussian_phz
 
-Intrinsic alignments
---------------------
-.. autofunction:: kappa_ia_nla
```

Comment:
I have reverted https://github.com/glass-dev/glass/pull/21 as I believe it is broken/needs testing. Don't want to waste time porting.